.circleci/config.yml

```diff
@@ -157,6 +157,7 @@ executors:
       PAYPAL_SANDBOX: true
       FXA_MX_RECORD_EXCLUSIONS: restmail.dev.lcip.org
       # Needed for react conversion playwright tests
+      ROLLOUT_GENERALIZED_REACT_APP: 1
```

Comment:
This will apply to all react routes that are on but not set to full prod rollout - currently that is only react email-first routes.